variants/vmware-k8s-1.21/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
```

Comment:
If we're adding VMware, we should also update QUICKSTART-VMWARE.md, at least for grammar around multiple VMware variants, but possibly also to make 1.21 the new de facto default.